### DISCUSSION TITLE
How about a PC version

### DISCUSSION THREAD

**Discussion by Peace-master2930** [z-huang/music#1246] [Ideas]
I love the app but also want to list to the songs in my laptop , so If you could make a PC version it would be great.

**Comment by kaneryu** [comment]
I'm working on one! It's still in development though, a few more months.

**Comment by SureshTimma** [comment]
> I'm working on one! It's still in development though, a few more months.

@KaneryU please add CSV playlist import option

**Reply by kaneryu** [reply]
Can you send me an example? I can implement that.

**Reply by SureshTimma** [reply]
> Can you send me an example? I can implement that.

@KaneryU I have checked this option in Rimusic App. This app also forked from VI Music.

YouTube may not able to understand from my words clearly. Please visit app once.

![Screenshot_2024-03-21-08-49-06-84_91b67804b5fb3353a77fb824d6bb0b50.jpg](https://github.com/z-huang/InnerTune/assets/148480722/92b38aad-c644-4e5a-8285-8016bc1fc35d)

**Reply by SureshTimma** [reply]
> Can you send me an example? I can implement that.

Do you want to work on app again??

**Reply by kaneryu** [reply]
> Do you want to work on app again??

Unfortunately I don't know the languages the app uses, so I can't work on it. The main app dev right now is @z-huang

**Comment by Tandoorie** [comment]
Hey, is there any movement on this one at all? We all really appreciate your work and this is a lifesaver!

**Comment by Der-Oelmann** [comment]
I can use innertunes on my pc already
